test/unit/language-test.js

```diff
@@ -353,18 +343,23 @@ describe('languageUtils', function() {
         });
 
         it('should only use custom localization block if "forceLocalizationDefaults" is true', function() {
-            sandbox.stub(Language, 'getLanguage').returns('fr');
```

Comment:
@DanFerrer why did you remove the stubbing ? If we don't stub then how are we verifying that we override the language when `forceLocalizationDefaults` is true ?

Reply:
I think you might have misunderstood our conversation from friday. What i meant to say was that `expect(Language.getLanguage()).to.equal('fr');` is unnecessary since get language is being stubbed and we don't need to unit test the stubbing. But in order to properly test that `forceLocalizationDefaults ` overrides the language, we need to stub `getLanguage`, makes sense ?